src/main/java/org/jabref/gui/contentselector/ContentSelectorDialog.java

```diff
@@ -302,8 +310,7 @@ private void applyChanges() {
 
         // Update all selectors in the current BasePanel.
         if (changedFieldSet) {
-            // We have added or removed content selectors, update the entry editor
-            panel.rebuildAllEntryEditors();
+            // TODO: We have added or removed content selectors, update the entry editor
```

Comment:
Ok, I get it. If this is just a false marking by github and the TODOs have been there before, then this can just move forward.

However, we need to be sure that we do not release 4.0 without content selectors. Should we open an issue for this? Or will you keep this in mind and open a PR at some point?